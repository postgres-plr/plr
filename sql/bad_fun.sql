-- should error out but should not crash PG
CREATE OR REPLACE
FUNCTION r_bad_fun()
RETURNS int4 AS
$BODY$
  deadbeef <- function(,bad) {}
  42
$BODY$
LANGUAGE plr;

SELECT r_bad_fun();
