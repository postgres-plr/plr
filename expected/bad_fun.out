-- should error out but should not crash PG
CREATE OR REPLACE
FUNCTION r_bad_fun()
RETURNS int4 AS
$BODY$
  deadbeef <- function(,bad) {}
  42
$BODY$
LANGUAGE plr;
SELECT r_bad_fun();
ERROR:  R interpreter parse error
DETAIL:  Error: bad value
CONTEXT:  In PL/R function r_bad_fun
